Fixes #179 : Travis build failing

CloudyPadmal opened the pull request:
Fixed the build fail due to Travis update recently!

Thanks @jithinbp for the guide and info!

I've cloned the exisiting pslab-desktop-apps repo and edited travis.yml to test the build. Find it [here](https://travis-ci.org/CloudyPadmal/testPSLAB-Desktop-Apps/jobs/273757008)

CloudyPadmal commented:
Python 2.7 version build passing and Python 3.4 version build failing! I am working on the failing part!

jithinbp commented on .travis.yml:
```diff
@@ -53,9 +54,9 @@ install:
-    - cp -R /usr/local/lib/python2.7/dist-packages/PSL /home/travis/virtualenv/python3.3.6/lib/python3.3/site-packages/
-    - cp -R /usr/local/lib/python2.7/dist-packages/PSL /home/travis/virtualenv/python3.4.4/lib/python3.4/site-packages/     
+    - if [[ $TRAVIS_PYTHON_VERSION == 2.7 ]]; then cp -R /usr/local/lib/python2.7/dist-packages/PSL /home/travis/virtualenv/python2.7.13/lib/python2.7/site-packages/; fi
+    - if [[ $TRAVIS_PYTHON_VERSION == 3.3 ]]; then cp -R /usr/local/lib/python2.7/dist-packages/PSL /home/travis/virtualenv/python3.3.6/lib/python3.3/site-packages/; fi
```
In lines 7, 12 only Python versions 2.7 and 3.4 are being used, so 3.3 either needs to be enabled, or this line removed

jithinbp commented:
@CloudyPadmal 3.4 build is failing because `python --version` returns `Python 3.4.6`, and pslab-python files are being copied to `3.4.4`, a directory which doesn't exist

CloudyPadmal commented:
Hi @jithinbp ! I've changed the python folder names to `3.4.6` and removed `3.3` check. Now the build is passing. :D Thanks for the support! :1st_place_medal:

CloudyPadmal commented:
Thanks all! I will merge this so that I can rebuild #178 !! :)
